lib/src/main/scala/org/typelevel/scalacoptions/ScalacOptions.scala

```diff
@@ -469,10 +475,7 @@ private[scalacoptions] trait ScalacOptions {
     * Enabled by default in 2.13.0+ and no longer accepted by the compiler as an option.
     */
   val privatePartialUnification =
-    privateOption(
-      "partial-unification",
-      version => version.isBetween(V2_11_11, V2_13_0)
-    )
+    privateOption("partial-unification", version => version.isBetween(V2_11_11, V2_13_0))
```

Comment:
Just to clarify: is this change supposed to be about re-formatting only?

Reply:
+1 on this. If it's just reformatting please revert it.

Reply:
Adressed